tools/ld-chroma-decoder/palcolour.cpp

```diff
@@ -372,6 +374,36 @@ void PalColour::detectBurst(LineInfo &line, const quint16 *inputData)
     line.bq /= burstNorm;
 }
 
+// Perform analog-style noise coring.
+void PalColour::doYNR(double *Yline)
+{
+    // High-pass filter for Y
+    auto yFilter(f_nr);
```

Comment:
f_nr is designed for the NTSC sample rate and luma bandwidth. Would it be better to have two separate filters in filtermaker for this?